guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,517 @@
 {
```

Comment:
<ul><li>Fixes incorrect format used in the request sent with a&nbsp;<code>dimension_values</code>&nbsp;argument for ArcGIS Online. </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<ul><li>updated the note!</li></ul>